staging/src/k8s.io/apimachinery/pkg/util/errors/errors.go

```diff
@@ -147,6 +147,24 @@ func Flatten(agg Aggregate) Aggregate {
 	return NewAggregate(result)
 }
 
+// UniqueAggregateWithCount removes duplicates from flattened Aggregate
+// and returns count for each unique error present in the Aggregate
+func UniqueAggregateWithCount(agg Aggregate) Aggregate {
+	if agg == nil || len(agg.Errors()) == 0 {
+		return agg
+	}
+	errs := Flatten(agg).Errors()
+	errorCountMap := map[string]int{}
+	for _, err := range errs {
+		errorCountMap[err.Error()]++
+	}
+	result := make([]error, 0, len(errorCountMap))
+	for errStr, count := range errorCountMap {
+		result = append(result, fmt.Errorf("%v(Count=%v)", errStr, count))
```

Comment:
I would at least put a space in between the error text and the count information. I wonder if `(Count=n)` is the best way to show this, or if something like `(repeated n times)` would look better? WDYT @timothysc @derekwaynecarr?

Also, if we keep the `Count=` format, I would make it lowercase `count`.